### DISCUSSION TITLE
Marvel Comics to Studios

### DISCUSSION THREAD

**Discussion by sujaldev** [browserengineering/book#337] [Ideas]
A little background to the people/things I'll refer to here:
- [`3b1b`](https://www.youtube.com/c/3blue1brown) : This a channel made by Grant Sanderson who makes videos explaining a variety of topics with animation.
- Now this person created a library called [`manim`](https://github.com/3b1b/manim) to make animations with python which led to a community maintained [`fork`](https://www.manim.community/) of this library.

The Idea:
How about videos styled like 3b1b's videos made with manim itself to compliment the book.
Why this would be great:
- Videos are created programmatically and so they can be made open source
- Some things could be conveyed more clearly with an animation maybe?
- Don't have much to support the statement but I have observed more people would be interested in a video course
- I don't think there exists any other resource like this on youtube so this would be the first of its kind

Me and my [friend](https://github.com/usyntest) are willing to start this idea if you permit.

What do you think?

**Comment by pavpanchekha** [comment]
Well, I certainly don't mind folks making videos based on our book. For our parts, Chris and I have been working on some interactive widgets that could be embedded into the book itself; there are some basic ones in Chapters 2 and 3. Honestly, I've found that building interactive widgets like this is a lot of work—mostly not due to the effort of coding them, but the effort to actually come up with nice visual explanations of the concepts in the book.
